packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -1223,6 +1223,66 @@ void main() {
 
     expect(item0, findsNothing);
   });
+
+  testWidgets('SliverReorderableList auto scrolls speed is configurable', (WidgetTester tester) async {
```

Comment:
@chunhtai may have thoughts on how to best test this.

Reply:
I think you can check the scrollcontroller.offset directly.

Given two reorderablelist with same setup but different scalar, if they are drag by the same distance and duration, they should scroll distances in the same ratio of the scalar

Reply:
Done